### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by SFarida**
### In this PR I have improve on the styles and made the site responsive

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -154,4 +152,58 @@ input::placeholder {
...
+  .chapter {
+    margin-bottom: 2rem;
+    align-items: center;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .list-item {
+    margin: 2.313rem 1.25rem 0;
+  }
```
Comment (jerryowusu) [inline src/styles/index.css]: There are some differences in the design in terms of the positioning and alignment of the elements in each book components as shown below :point_down: 

The elements in each book component such as the book title and author, the progress icon,  the chapter information, and the update progress bar are supposed to be aligned horizontally but yours is aligned vertically. Kindly check and fix this anomaly to make your project match the actual design. 

### Your design 

Screenshot from 2023-03-04 13-22-26


### The actual design 

Screenshot from 2023-03-04 13-07-07

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @SFarida, 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

nss-nichole


## Highlights
Great effort so far :+1: 
No linter errors :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/index.css (hunk 1 above)
Comment (SFarida) [reply src/styles/index.css]: Hello @jerryowusu this happens on screens width equal to or smaller than 1199px. I added this for responsiveness as you can see on line 184. On a larger screen the design is the same, can you please check again or do you suggest I remove or reduce the break point?

Comment (SFarida) [reply src/styles/index.css]: I just reduced the break point to 991px I hope its ok

File: src/styles/App.css [hunk 2]
```diff
@@ -1,9 +1,9 @@
 .App {
-  width: 87.5rem;
+  /* width: 87.5rem; */
```
Comment (huseyinbiyik) [inline src/styles/App.css]: [OPTIONALK] Please don't leave any uncommented code in your codebase. It is better to git history if you need the specific code written before. In this way, you will keep your codebase neat, organized and professional.

**Review by huseyinbiyik — APPROVED** [review]
Hi @SFarida,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

- Since I can't select those lines I am adding here as screenshot. The design looks great, but you have too many repetitive CSS rules. So to DRY them you can use class attributes or add general rules in the beginning of CSS file. Check [these practices](https://speckyboy.com/good-bad-css-practices/) to learn more.
Screenshot 2023-03-04 at 5 05 03 PM

- README looks awesome but let's remove this templated text from your README file to make it more awesome 😮 
Screenshot 2023-03-04 at 5 05 46 PM


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**